src/interface/client.js

```diff
@@ -34,9 +30,22 @@ export function request(options) {
 
             options.window = el.contentWindow;
 
-            if (!options.window) {
-                throw new Error('Expected options.window');
+            
+        } else if (options.window instanceof HTMLElement) {
+            
+            if (options.window.tagName.toLowerCase() !== 'iframe') {
+                throw new Error(`Expected options.window ${options.window} to be an iframe`);
+            }
+
+            if (!options.window.contentWindow) {
```

Comment:
Let's also add this check when a string is passed for `options.window` to cover all of our bases?